allennlp/training/trainer.py

```diff
@@ -747,6 +747,11 @@ def train(self) -> Dict[str, Any]:
             self._save_checkpoint(epoch, validation_metric_per_epoch, is_best=is_best_so_far)
             self._metrics_to_tensorboard(epoch, train_metrics, val_metrics=val_metrics)
             self._metrics_to_console(train_metrics, val_metrics)
+            for ind, param_group in enumerate(self._optimizer.param_groups):
```

Comment:
`ind` is for `index`?  If so, just use `index` - for the price of just two characters, I don't have to guess what you mean ;).